package.json

```diff
@@ -0,0 +1,45 @@
+{
+  "name": "@react-native-common-modules/source",
+  "version": "0.0.1",
+  "license": "MIT",
+  "scripts": {
+    "build": "nx build",
+    "test": "nx test"
+  },
+  "dependencies": {
+    "@pietile-native-kit/keyboard-aware-scrollview": "^1.5.0",
+    "react": "^18.2.0",
+    "react-native": "^0.74.1",
+    "react-native-safe-area-context": "^4.10.3",
+    "tslib": "^2.3.0"
+  },
+  "devDependencies": {
+    "@nx/eslint": "19.1.1",
+    "@nx/eslint-plugin": "19.1.1",
+    "@nx/js": "19.1.1",
+    "@nx/vite": "19.1.1",
+    "@nx/web": "19.1.1",
```

Comment:
Not sure we need these dependencies. What the template have you used?
I think we could use the [NX Expo template](https://nx.dev/nx-api/expo) and have:
- some demo app to see and test the components in action. This app won't be published to NPM
- the `src/lib` with common modules will be a [publishable library](https://nx.dev/concepts/buildable-and-publishable-libraries#publishable-libraries) to distribute in over NPM

Reply:
Done